src/helpers.js

```diff
@@ -3,8 +3,100 @@ export const HELPERS_ID = '\0commonjsHelpers';
 export const HELPERS = `
 export var commonjsGlobal = typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
 
-export function commonjsRequire () {
-	throw new Error('Dynamic requires are not currently supported by rollup-plugin-commonjs');
+const realRequire = require;
```

Comment:
Any reason for making a copy here? We do this nowhere else and directly using require does not break any tests.